helm/templates/configmap.yaml

```diff
@@ -42,6 +42,8 @@ data:
   INGESTION_UPDATE_JOB_TYPE: {{ $jobManagement.ingestion.update.jobType | quote }}
   INGESTION_MERGE_TASK_TYPE: {{ $jobManagement.ingestion.update.mergeTilesTaskType | quote }}
   INGESTION_SWAP_UPDATE_JOB_TYPE: {{ $jobManagement.ingestion.updateSwap.jobType | quote }}
+  SEED_JOB_TYPE: {{ $jobManagement.seed.jobType | quote }}
```

Comment:
why do you have "seed" hirarechy here and and values and not in config? please keep the same and add seed hirerachy to config

Reply:
Becouse as mutual jobmanagment build up by this hirarechy, but the job&task configs on overseer is flatten so I kept the exists order.

Reply:
we decided to move it to the seed hierarchy.

Reply:
done